src/instrumentation/health_check.go

```diff
@@ -0,0 +1,66 @@
+package instrumentation
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	"gopkg.in/yaml.v3"
+)
+
+type Health struct {
+	Healthy            bool               `json:"healthy"`
+	Status             string             `json:"status"`
+	StartTime          string             `json:"start_time_unix_nano"`
+	StatusTime         string             `json:"status_time_unix_nano"`
+	AgentRunID         string             `json:"agent_run_id"`
```

Comment:
`agentRunId` is no longer in the spec so it can be removed: https://source.datanerd.us/agents/agent-specs/blob/main/super_agent/Health-Check.md

Reply:
fixed